### ISSUE TITLE
Forcing Friends causes friend list error

### ISSUE THREAD

**Issue by QuinnBast** [ShawnClake/UserExtended#117]
In the friends module I added two users as friends.

I logged into one of the users accounts and clicked on their friends list to see if it worked. I got the following error:

![image](https://cloud.githubusercontent.com/assets/22599103/25455169/0ce12606-2a8c-11e7-8dea-23c4cd8dc08c.png)

**Comment by QuinnBast** [comment]
Update: I sent a request to him from a different account and manually accepted it. Going into the "friends list" shows one account as a friend but not the account that I forced friends in the friend manager.

![image](https://cloud.githubusercontent.com/assets/22599103/25455438/137e1b26-2a8d-11e7-8407-a7b3bc9e57fa.png)

(The top one was added in the friends module. The second one was added manually sending the request.)

![image](https://cloud.githubusercontent.com/assets/22599103/25455481/36c52804-2a8d-11e7-9d67-0304151073ea.png)

After deleting the manually added friend it shows the friend list with no error

![image](https://cloud.githubusercontent.com/assets/22599103/25455536/67feccae-2a8d-11e7-8860-4a3fc6f24d65.png)

![image](https://cloud.githubusercontent.com/assets/22599103/25455552/7580a7bc-2a8d-11e7-9018-61969213c0b1.png)
